test/EventStore.Client.Tests.Common/EventStoreClientFixtureBase.cs

```diff
@@ -159,7 +159,7 @@ protected class EventStoreTestServer : IAsyncDisposable {
 				};
 			}
 
-			public Task Start(CancellationToken cancellationToken = default) => _container.Start(cancellationToken);
+			public Task Start(CancellationToken cancellationToken = default) => Task.CompletedTask;//_container.Start(cancellationToken);
```

Comment:
This line can be safely reverted.
```suggestion
			public Task Start(CancellationToken cancellationToken = default) => _container.Start(cancellationToken);
```